### PR TITLE
solarwinds logo added

### LINKED ISSUE #170: [Site] Add SolarWinds Logo
**Current Behavior**
Individuals at SolarWinds have contributed the Service Mesh Performance initiative.

**Desired Behavior**
SolarWinds' participation should be represented in the Contributors section of the website.

Find a copy of the SolarWinds logo on the https://meshery.io site.

---
**Contributor Resources**
- [Service Mesh Performance documentation source](https://github.com/layer5io/service-mesh-performance/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/service-mesh-performance/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by soham4abc**
Signed-off-by: Soham Banerjee <sohambanerjee4abc@hotmail.com>

**Description**
Solarwinds logo added and svg file included 

This PR fixes #170 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by soham4abc** [comment]
@leecalcote  Please review!

File: docs/_includes/contributors.html [hunk 1]
```diff
@@ -55,10 +55,14 @@ <h1 class="section-heading text-uppercase">{{ site.data.sitetext.contributors.ti
     <div class="client-slide fiserv1">
 			<img src="/assets/img/contributors/fiserv.svg" height="50" width="120" alt="Fiserv" />
 		</div>
+	<div class="svg-space"></div>
```
Comment (geraldmaboshe) [inline docs/_includes/contributors.html:58]: Could this code be formatted so it's easier to read?

Comment (adithyaakrishna) [reply docs/_includes/contributors.html:58]: @soham4abc Any update on this?

Comment (soham4abc) [reply docs/_includes/contributors.html:58]: Do you want me to remove the extra spaces?

Comment (soham4abc) [reply docs/_includes/contributors.html:58]: Do you want me to remove the spaces?

**Comment by soham4abc** [comment]
@adithyaakrishna  Pls do have a look. And do let me know if any changes are required

**Comment by adithyaakrishna** [comment]
@soham4abc Can you please check the deploy preview as the solarwinds logo isn't seen 👀

**Comment by soham4abc** [comment]
Closing this PR There are some issues.. Will open after completing

**Pull request closed without merging**